### ISSUE TITLE
Attribute role="presentation" is affected by tabindex attributes for JAWS 2018

### ISSUE THREAD

**Issue by githubdam** [FreedomScientific/standards-support#106]
### Summary

JAWS 2018 reads the first input field as "Table Column 2 Row 1" followed by the actual field name. However, when the tabIndex attribute is removed on both of the outer `<div>` tags, JAWS 2018 doesn't read the tabular structure just the field name. 

### Expected result

>JAWS 2018 reads the first input field as "Table Column 2 Row 1" followed by the actual field name. 

### Actual result

>  JAWS reads the first input field using its accessible name without mention of the tabular structure it's contained in. 

### Example
```
<!Doctype hmtl>
<html>

<head>
    <title>Simple test on role presentation with tabindex</title>
</head>
<div tabindex="0">
    <div tabindex="-1">
        <form>
            <h1 title="Search">Search</h1>
            <span tabindex="0" title="Toggle" role="button" aria-expanded="true">&nbsp;<span>Toggle Button</span></span>
            <table summary="Search Criteria" role="presentation">
                <tbody>
                    <tr>
                        <th>First Name</th>
                        <td><input title="First Name" type="text" value=""></td>
                        <th>Surname</th>
                        <td><input title="Surname" type="text" value=""></td>
                    </tr>
                </tbody>
            </table>
        </form>
    </div>
</div>

</html>
```

#### JAWS version and build number
JAWS 2018.1807

#### Operating System and version
Windows 7 64bit

#### Browser and version:
Chrome Version 68.0.3440.106 (Official Build) (64-bit)

**Comment by wardav** [comment]
Getting the same behavour in JAWS 2019. Seeing it where the page has a `tabindex` value on a parent container, such as for a skip link. For example:

```
<div tabindex="-1">
...
<table role="presentation">
...
</table>
...
</div>
```

Is incorrectly parsed as a table.

**Comment by lamador** [comment]
@wardav Please provide browser name and complete code sample.

**Comment by jducrot** [comment]
This is not a bug because the role of presentation strips the element of its semantic meaning. As stated in the Aria 1.1 spec "For any element with a role of presentation and which is not focusable, the user agent MUST NOT expose the implicit native semantics of the element"

**Comment by wardav** [comment]
Sorry for not adding more information. To confirm, my case was that the use of `tabindex` prompted JAWS to ignore the use of `role="presentation"`. 

It would correctly ignore the table semantics on.

```
<div>
  <table role="presentation">
    ...
  </table>
</div>
```

But it would incorrectly infer that this was a data table *despite* the `role="presentation"`.

```
<div tabindex="-1">
  <table role="presentation">
    ...
  </table>
</div>
```

I can see that Scott has logged this as an IE but, which makes sense. Let me know if you would like me to build a CodePen and test this with a few versions of JAWS and various browsers.

**Comment by JAWS-test** [comment]
With JAWS 2020, the problem of outputting a table with role=presentation as a table when there is a tabindex at the enclosing div no longer occurs (in IE 11, Firefox, Chrome)

**Comment by JAWS-test** [comment]
In IE 11, it only occurs with JAWS 2020 if the table contains th element. My initial test had only td element (see previous comment). 
With th, the problem does not occur in Chrome and Firefox.

**Comment by scottaohara** [comment]
@jducrot, @wardav  the issue as originally described does not occur when testing with latest Chrome and JAWS 2020.  However, i was able to reproduce the issue with IE11, where if a layout table (`table role=presentation`) is nested within a `div tabindex=-1` then the `role=presentation` on the `table` will be ignored, and *incorrectly* exposed as a `table` only with JAWS 2020.2004.66, JAWS 2020.2006.12 and IE11, per the following test case [test case with results](https://scottaohara.github.io/tests/aria-none).   

Adding a new test after @JAWS-test's latest comment, this issue can reproduced even when a table contains no `th` elements, as long as there are enough cells in the table.


From what I can see when reviewing the exposed roles by the platform APIs here, is that IE11 continues to expose the tables with a table role, regardless of `role=presentation`... but JAWS is putting in some extra effort to work around IE11's poor support, and correctly treating the second table in the test case as presentational.  Unfortunately it is getting tripped up due to the `tabindex=-1` wrapper around the other tables, which...

**Comment by wardav** [comment]
For what it is worth, I found this because someone was using `tabindex="-1"` on the `<main>` container -- I presume to aid the skip link -- and then the `<main>` region had a layout table with`role="presentation"`.

It didn't need to be a direct child of a `<div>` with `tabindex`. This happened by virtue of the page region having the `tabindex` and the layout table was a distant ancestor.

**Comment by scottaohara** [comment]
that is helpful, thank you @wardav

**Comment by scottaohara** [comment]
cc @BenKeyFSI & @kurtjwolf to put this on your radars for any internal bugs that may already exist for something like this / if there's anything that can be done for this IE11 specific issue.

**Comment by stevefaulkner** [comment]
legacy IE bug - wontfix
